worker/src/core/loaders/message-worker/govsg.class.ts

```diff
@@ -72,6 +72,7 @@ class Govsg {
         body: string
         campaignId: number
         whatsappTemplateLabel: string
+        language: string
```

Comment:
Note: This is actually language code but the downstream function looks for `language` field. I don't intend to refactor anything.